### PR TITLE
Class 4,6,7,8 PR request.

### PR DESCRIPTION
**Pull request by fahimar**
I will update all work of class 4,6,7,8 within 3/4 days.

### CODE REVIEW

**Review by tajnur007 — APPROVED** [review]
Approving this PR but you should do more practice on Class-8. There have more crucial part you didn't practiced yet. So you can do it in another PR or you can do it in this PR. If you do it into this PR then ping me once after finishing the practices.

PR Approved!

**Review by tajnur007 — APPROVED** [review]
PR Approved!

File: class-8/error-handling.js [hunk 1]
```diff
@@ -45,5 +45,40 @@ console.log("Welcome to Class-8");
 // const encodedURI = encodeURI("\uD800");
```
Comment (tajnur007) [inline class-8/error-handling.js]: Uncomment the coding part, just keep the actual comments.

Comment (fahimar) [reply class-8/error-handling.js]: Ok , I have done it.

**Pull request merged**